showcase/app/templates/layouts/grid.hbs

```diff
@@ -0,0 +1,355 @@
+      <Shw::Outliner>
+        <Hds::Layout::Grid @columnMinWidth="250px" @gap="12" as |HLG|>
+          <HLG.Item @colSpan="2">
+            <Shw::Placeholder @text="#1" @height="24" @background="#e4c5f3" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#2" @height="24" @background="#e5ffd2" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#3" @height="24" @background="#d2f4ff" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#4" @height="24" @background="#fff8d2" />
+          </HLG.Item>
+        </Hds::Layout::Grid>
+      </Shw::Outliner>
+    </SG.Item>
+
+    <SG.Item @label="colSpan=3">
+      <Shw::Outliner>
+        <Hds::Layout::Grid @columnMinWidth="250px" @gap="12" as |HLG|>
+          <HLG.Item>
+            <Shw::Placeholder @text="#1" @height="24" @background="#e4c5f3" />
+          </HLG.Item>
+          <HLG.Item @colSpan="3">
+            <Shw::Placeholder @text="#2" @height="24" @background="#e5ffd2" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#3" @height="24" @background="#d2f4ff" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#4" @height="24" @background="#fff8d2" />
+          </HLG.Item>
+        </Hds::Layout::Grid>
+      </Shw::Outliner>
+    </SG.Item>
+
+    <SG.Item @label="colSpan=4">
+      <Shw::Outliner>
+        <Hds::Layout::Grid @columnMinWidth="250px" @gap="12" as |HLG|>
+          <HLG.Item @colSpan="4">
+            <Shw::Placeholder @text="#1" @height="24" @background="#e4c5f3" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#2" @height="24" @background="#e5ffd2" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#3" @height="24" @background="#d2f4ff" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#4" @height="24" @background="#fff8d2" />
+          </HLG.Item>
+        </Hds::Layout::Grid>
+      </Shw::Outliner>
+    </SG.Item>
+  </Shw::Grid>
+
+  <Shw::Text::H3>rowSpan</Shw::Text::H3>
+
+  <Shw::Grid @columns="1" @gap="1.5rem" as |SG|>
+    <SG.Item @label="rowSpan=2">
+      <Shw::Outliner>
+        <Hds::Layout::Grid @columnMinWidth="33.33%" @gap="12" as |HLG|>
+          <HLG.Item @rowSpan="2">
+            <Shw::Placeholder @text="#1" @height="100%" @background="#e4c5f3" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#2" @height="24" @background="#e5ffd2" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#3" @height="24" @background="#d2f4ff" />
+          </HLG.Item>
+          <HLG.Item>
+            <Shw::Placeholder @text="#4" @height="24" @background="#fff8d2" />
+          </HLG.Item>
+        </Hds::Layout::Grid>
+      </Shw::Outliner>
+    </SG.Item>
+
+    <SG.Item @label="rowSpan=3">
```

Comment:
[idea] I would consider having two examples here, that are both interesting: 
- one where it's the first item that has `rowSpan=3`
- one where it's the second item that has `rowSpan=3`